pkg/util/vpc_nat_gateway.go

```diff
@@ -2,10 +2,17 @@ package util
 
 import "fmt"
 
+var (
```

Comment:
how about using a const?

Reply:
Allows for changing the prefix on the fly from other packages, such as the one managing the configmap for the nat gateway

Reply:
ok